code/datums/greyscale/_greyscale_config.dm

```diff
@@ -306,4 +306,38 @@
 	output["icon"] = GenerateBundle(colors, debug_steps)
 	return output
 
+/datum/greyscale_config/proc/Generate_universal_icon(color_string, target_bundle_state, datum/universal_icon/last_external_icon)
+	return GenerateBundle_universal_icon(color_string, target_bundle_state, last_external_icon=last_external_icon)
+
+/// Handles the actual icon manipulation to create the spritesheet
+/datum/greyscale_config/proc/GenerateBundle_universal_icon(list/colors, target_bundle_state, datum/universal_icon/last_external_icon)
```

Comment:
why is there zero consistency with the proc naming. please fix this.

Reply:
the consistency is a combination of the old functions (GenerateInternal...) and iconforge (which uses snake case literally everywhere)

Would you rather it be `GenerateUniversalIcon`?

Reply:
basically the reason that they're like this is because I just copied the old functions but made them output universal icons so they should match the original function names